workflows.yml

```diff
@@ -114,30 +114,7 @@ ParsePdr:
           - States.ALL
           ResultPath: '$.exception'
           Next: StopStatus
-      Next: CheckStatus
-    CheckStatus:
```

Comment:
These steps should return once https://bugs.earthdata.nasa.gov/browse/CUMULUS-392 has been fixed